### PR TITLE
docs(zod-openapi): add components registoration and auth configutation tips

### PR DESCRIPTION
**Pull request by sor4chi**
related: #261 

Since the guide was missing, we added a guide on authentication to the document to prevent the same question from occurring.

### CODE REVIEW

File: packages/zod-openapi/README.md [hunk 1]
```diff
@@ -284,6 +284,48 @@ const appRoutes = app.openapi(route, (c) => {
 const client = hc<typeof appRoutes>('http://localhost:8787/')
 ```
 
+## Tips
+
+### How to register components
+
+You can register components to the registry as follows:
```
Comment (yusukebe) [inline packages/zod-openapi/README.md:291]: You have to add a link to:
https://github.com/asteasolutions/zod-to-openapi#defining-custom-components

**Comment by yusukebe** [comment]
Hi @sor4chi 

Thanks for the PR. Almost good but I've left a comment.

File: packages/zod-openapi/README.md (hunk 1 above)
Comment (sor4chi) [reply packages/zod-openapi/README.md:291]: Done!

Comment (yusukebe) [reply packages/zod-openapi/README.md:291]: Good!

**Review by yusukebe — APPROVED** [review]

**Pull request merged**